pxf/HivePXF.html.md.erb

```diff
@@ -495,24 +511,24 @@ postgres=# SELECT * FROM hcatalog.default.sales_info;
 
 To obtain a description of a Hive table with HCatalog integration, you can use the `psql` client interface.
 
--   Within HAWQ, use either the `\d                                         hcatalog.hive-db-name.hive-table-name` or `\d+                                         hcatalog.hive-db-name.hive-table-name` commands to describe a single table. For example, from the `psql` client interface:
+-   Within HAWQ, use either the `\d                                         hcatalog.hive-db-name.hive-table-name` or `\d+                                         hcatalog.hive-db-name.hive-table-name` commands to describe a single table. `\d+` displays both the HAWQ and Hive data type, while `\d` displays only the HAWQ data type. For example, from the `psql` client interface:
```

Comment:
\d feature was designed to not stick to any particular source. In above example it's Hive, but it might be something else in future.